lib/active_model/serializable_resource.rb

```diff
@@ -1,7 +1,8 @@
 require 'set'
 module ActiveModel
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter,
+                                   :jsonapi_toplevel_meta])
```

Comment:
My thought was that people might (although ti wouldn't be great) want to handle two APIs with one app, maybe for migrating to a newer version of the standard or something.